### PR TITLE
Fix inconsistent colors

### PR DESCRIPTION
**Pull request by XanderD99**
### Thanks a lot for contributing!<br>

Found some more inconsistencies with the background and text colors, so fixed them.

Also added a factory to TalkerScreenTheme where you can just pass your ThemeData and optionally custom log colors.

In .vsocde I left my talker.code-workspace. I find this easy to use in a nested packages case.

### CODE REVIEW

File: packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart [hunk 1]
```diff
@@ -30,14 +32,23 @@ class TalkerScreenTheme {
     }
     return _defaultColors;
   }
+
+  factory TalkerScreenTheme.fromTheme(ThemeData theme, [LogColors? logColors]) {
+    return TalkerScreenTheme(
+      backgroundColor: theme.colorScheme.background,
+      textColor: theme.colorScheme.onBackground,
+      cardColor: theme.colorScheme.surface,
+    );
```
Comment (Frezyx) [inline packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart:41]: Here we need to provide logColors argument

File: packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart [hunk 2]
```diff
@@ -30,14 +32,23 @@ class TalkerScreenTheme {
...
 
-extension MapTalkerFlutterColorsExt on Map<TalkerLogType, Color> {
+extension MapTalkerFlutterColorsExt on LogColors {
   Color getByType(TalkerLogType type) {
     return this[type] ?? Colors.grey;
   }
 }
 
+// 💡 why not merge this into
 const _defaultColors = {
```
Comment (Frezyx) [inline packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart:52]: Sorry I don't got you 🤷‍♂️

**Review by Frezyx — CHANGES_REQUESTED** [review]
Hello @XanderD99 👋
Thanks a lot for this contribution! 👍

I will merge changes after review fixes

File: packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart (hunk 2 above)
Comment (XanderD99) [reply packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart:52]: You can ignore this, forgot to remove it 😅

File: packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart (hunk 1 above)
Comment (XanderD99) [reply packages/talker_flutter/lib/src/ui/theme/talker_screen_theme.dart]: fixed 👍

**Comment by Frezyx** [comment]
Hello @XanderD99!
Thank you for your fixes!

**Pull request merged**